tests/Doctrine/Tests/DBAL/Platforms/SQLServer2008PlatformTest.php

```diff
@@ -19,4 +19,9 @@ public function testGeneratesTypeDeclarationForDateTimeTz()
                 array())
         );
     }
+
+    public function getGenerateUniqueIndexSql()
```

Comment:
Sorry, I thought that SQLServer2012PlatformTest was inheriting from SQLServer2008PlatformTest. Corrected in https://github.com/j-d/dbal/commit/4ad0c2abf568a6770b44833a2205cfbdd6b27982